doc/manuals/code_contributions.md

```diff
@@ -51,6 +51,59 @@ Thanks to Alejandro Villamarin (published around October 2013):
           System.err.println("Failed. Reason is " + e.getMessage());
        }
 
+## JavaScript and NodeJS
+SmartSDK team developed and [SDK Client](https://github.com/smartsdk/ngsi-sdk-javascript) compatible with NSGIv2. 
```

Comment:
A reference to the time in which code contribution has been produced should be included. E.g. (from other contributions in this document): "(published around June 2014)".